### PR TITLE
Add test for WebsocketDataHandler

### PR DESCRIPTION
**Pull request by realcalu**
// Describe your PR here; eg. Fixes #issueNo
Enhance shenyu-sync-data-websocket module unit test，make its coverage rate is more than 80% pr#1458

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/projects/soul/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by yu199195** [comment]
Hi， please check ci ,  ,welcome join!

File: shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package org.apache.shenyu.plugin.sync.data.websocket.handler;
```
Comment (yu199195) [inline shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java:1]: please add apache license  2.0 header

**Review by tydhot — COMMENTED** [review]
plz check the checkstyle

File: shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import org.apache.shenyu.sync.data.api.AuthDataSubscriber;
+import org.apache.shenyu.sync.data.api.MetaDataSubscriber;
+import org.apache.shenyu.sync.data.api.PluginDataSubscriber;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+
+public class WebsocketDataHandlerTest {
+    PluginDataSubscriber pluginDataSubscriber;
```
Comment (yu199195) [inline shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java:22]: add private  and maybe have empty line

File: shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    WebsocketDataHandler websocketDataHandler;
+
+    @Before
+    public void testWebsocketDataHandler() {
+        pluginDataSubscriber = mock(PluginDataSubscriber.class);
+
```
Comment (yu199195) [inline shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java:28]: remove empty line

File: shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java (hunk 2 above)
Comment (realcalu) [reply shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java]: I have add private and remove the empty line

File: shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java (hunk 1 above)
Comment (realcalu) [reply shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java:17]: I have add license

File: shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java (hunk 3 above)
Comment (realcalu) [reply shenyu-sync-data-center/shenyu-sync-data-websocket/src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java]: I have remove empty line

**Comment by realcalu** [comment]
I have been add the licenses and remove the empty line

**Comment by yu199195** [comment]
@realcalu 

Error:  src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java:[1] (misc) NewlineAtEndOfFile: File does not end with a newline.
Error:  src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java:[17] (whitespace) EmptyLineSeparator: 'package' should be separated from previous statement.
Error:  src/test/java/org/apache/shenyu/plugin/sync/data/websocket/handler/WebsocketDataHandlerTest.java:[37] (whitespace) EmptyLineSeparator: 'CLASS_DEF' has more than 1 empty lines before.


please  checked it

**Comment by realcalu** [comment]
sorry! I haven been run "mvn clean install -Dmaven.javadoc.skip=true" to check the CI and I have been fixed the code format style.

**Comment by yu199195** [comment]
Hi welcome join,  please add  wechat  xixy199195

**Pull request merged**